std-bits/base/src/main/java/org/enso/base/spi/EnsoServiceLoader.java

```diff
@@ -0,0 +1,65 @@
+package org.enso.base.spi;
+
+import java.util.List;
+import java.util.ServiceLoader;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import org.graalvm.polyglot.Value;
+
+/** A helper class that allows loading Enso services from the classpath. */
+public class EnsoServiceLoader<T extends EnsoService> {
+  private final ServiceLoader<T> loader;
+  private transient List<T> cachedProviders = null;
+
+  protected EnsoServiceLoader(Class<T> clazz) {
+    loader = ServiceLoader.load(clazz, clazz.getClassLoader());
+  }
+
+  public static <T extends EnsoService> EnsoServiceLoader<T> make(Class<T> clazz) {
```

Comment:
There is [load](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/ServiceLoader.html#load(java.lang.Class)) in `ServiceLoader`. I don't see any need to be _"more inventive"_ and call similar method `make`.

Reply:
Ok